test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -1219,7 +1219,7 @@ public async Task Restore_WithHttpSource_Warns()
         [InlineData("", true)]
         [InlineData("true", false)]
         [InlineData("TRUE", false)]
-        public async Task Restore_WithHttpSourceAndFalseAllowInsecureConnections_WarnsCorrectly(string allowInsecureConnections, bool hasHttpWarning)
+        public async Task Restore_WithHttpSourceAndFalseAllowInsecureConnections_FailsCorrectly(string allowInsecureConnections, bool shouldFail)
```

Comment:
The name doesn't make sense to me. Is AllowInsecureConnections false, like the name says, or is it true, like the data says?

Reply:
I have made it into two tests.